common/changes/miwhea-link-focus-border_2017-01-05-01-19.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fix an issue where the focus border around a Link was positioned incorrectly.",
```

Comment:
This comment shows up in the changelog; please fix in this format:

Link: focus border is now positioned correctly when the link spans multiple lines.